src/sys/unix/waker.rs

```diff
@@ -276,7 +278,7 @@ mod pipe {
 }
 
 #[cfg(all(
-    mio_unsupported_force_poll_poll,
+    any(target_os = "solaris", mio_unsupported_force_poll_poll),
```

Comment:
Not need, you already added it below.

Reply:
I got following errir without this change:
```
error[E0432]: unresolved import `crate::sys::unix::waker::WakerInternal`
 --> src/sys/unix/selector/poll.rs:7:5
  |
7 | use crate::sys::unix::waker::WakerInternal;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `WakerInternal` in `sys::unix::waker`

error: unused import: `AsRawFd`
  --> src/sys/unix/selector/poll.rs:11:25
   |
11 | use std::os::unix::io::{AsRawFd, RawFd};
   |                         ^^^^^^^
   |
note: the lint level is defined here
  --> src/lib.rs:5:5
   |
5  |     unused_imports,
   |     ^^^^^^^^^^^^^^
```